modules/test/playwright/tests/frontend-taglib/fieldsetTaglib.spec.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {isolatedSiteTest} from '../../fixtures/isolatedSiteTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {journalPagesTest} from '../journal-web/fixtures/journalPagesTest';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	isolatedSiteTest,
+	journalPagesTest,
+	loginTest()
+);
+
+test(
+	'Tooltip should be translated correctly',
+	{
+		tag: '@LPD-43309',
+	},
+	async ({apiHelpers, journalPage, page, site}) => {
+		await test.step('Create a web content folder and access to edit it', async () => {
+			const webContentFolder =
+				await apiHelpers.jsonWebServicesJournal.addFolder({
+					groupId: site.id,
+				});
+
+			await journalPage.goto(site.friendlyUrlPath);
+
+			await journalPage.changeView('cards');
+
+			await page
+				.locator(`.card-body:has-text('${webContentFolder.name}')`)
+				.getByLabel('More actions')
+				.click();
```

Comment:
As it is, this test should be in `journal-web`, and sent for review to the product team. It uses a lot of web content specifics, that product team can change at any time.

Alternative is to add a sample of tag in `frontend-taglib-sample-web`, and add a test in something like
```
/frontend-taglib/tests/liferay-frontend/fieldset.spec.ts
```

Reply:
Thank you, I will be making changes to journal-web and sending to product team!